src/components/checkboxCollapsible/CollapsibleCheckbox.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { forwardRef, ReactNode } from 'react';
+import { Checkbox as ChakraCheckbox } from '@chakra-ui/react';
+
+import Flex from '../flex';
+import { CheckboxProps } from '../checkbox';
+
+export interface CollapsibleCheckboxProps extends CheckboxProps {
+  icon?: ReactNode;
+}
+
+const CollapsibleCheckbox = forwardRef<
+  HTMLInputElement,
+  CollapsibleCheckboxProps
+>(
+  (
+    {
+      name,
+      value,
+      isDisabled = false,
+      isChecked = false,
+      onChange,
+      label,
+      isInvalid,
+      isIndeterminate = false,
+      size = 'lg',
+      fontWeight = 'regular',
+      variant = 'alignCenter',
+      readOnly = false,
+      fullWidth = false,
+      icon,
+      ...props
+    },
+    ref,
+  ) => (
+    <Flex alignItems="center">
+      <ChakraCheckbox
+        {...props}
+        ref={ref}
+        name={name}
+        value={value}
+        isDisabled={isDisabled}
+        isChecked={isChecked}
+        onChange={onChange}
+        isInvalid={isInvalid}
+        isIndeterminate={isIndeterminate}
+        size={size}
+        fontWeight={fontWeight}
+        variant={variant}
+        readOnly={readOnly}
+        width={fullWidth ? 'full' : undefined}
+      >
+        {label}
+      </ChakraCheckbox>
```

Comment:
this doesn't allow us to pass facets as in the design here: 
![Bildschirmfoto 2024-02-06 um 08 24 02](https://github.com/smg-automotive/components-pkg/assets/71456764/33e7c0e9-ce0b-4613-959c-6bca0599455a)

I think we need some mix between this component you have here and `CheckboxFilter`. Maybe it's possible to extract the checkbox part of it and use it here?